index.html

```diff
@@ -11,11 +11,13 @@
 </head>
 <body>
     <header>
+      <nav>
             <a href="">
                 <h4>Ebuka Nweje</h4>
             </a>
-          <nav>
+          
             <ul class="items">
+              <span class="close">X</span>
               <li class="item"><a href="#" class="link">Portfolio</a></li>
               <li class="item"><a href="#" class="link">About</a></li>
               <li class="item"><a href="#" class="link">Contact</a></li>
```

Comment:
- [ ] Kindly check on your menu options When the user clicks (or taps) any of the mobile menu options, its suppose to take the user to the correct part of the page, and also the mobile menu disappears.
- [ ] [optional] Kindly check on your mobile version , it differs with the Figma design:

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/174876255-46008273-36a6-4b57-9b1a-45d273837ffc.png)  |  ![image](https://user-images.githubusercontent.com/61408860/174875989-80e3aaa8-dbc8-4bba-b2cb-530cee1cd890.png)